CONTRIBUTING.md

```diff
@@ -608,6 +585,24 @@ Redoc generates HTML documentation using the InfluxDB `swagger.yml`.
 For more information about generating InfluxDB API documentation, see the
 [API Documentation README](https://github.com/influxdata/docs-v2/tree/master/api-docs#readme).
 
+## InfluxDB URLs
+InfluxDB and InfluxDB cloud are accessed at different and varying URLs.
+The InfluxDB documentation customizes InfluxDB URLs inside of code blocks using
+the product / region selected by the user.
+InfluxDB URLs are configured in `/data/influxdb_urls.yml`.
+
+The default URL that is replaced inside of code blocks is `http://localhost:9999`.
```

Comment:
By default, the URL in code blocks is `http://localhost:9999` for InfluxDB OSS. 

Maybe include an example? If, for example, a user selects the InfluxDB Cloud product and the US West (Oregon) region, URLs throughout the documentation are automatically updated to `https://us-west-2-1.aws.cloud2.influxdata.com`